source/Junk.js

```diff
@@ -0,0 +1,32 @@
+var Pixi = require("pixi.js")
+var Utility = require("./Utility")
+
+import Reference from "./Reference.js"
+import Junkership from "./Junkership.js"
+import Trashbot from "./Trashbot.js"
+import Score from "./Score.js"
+
+export default class Junk extends Pixi.Sprite {
+    constructor(x, y) {
+        super(PIXI.loader.resources.junk.texture)
+        this.position.x = x
+        this.position.y = y
+        this.despawn = Date.now()
+    }
+
+    update(delta) {
+        if (Date.now() - this.despawn >= 5000) {
+            game.removeChild(this)
+            this.destroy()
+        } else
+            game.children.forEach((child) => {
+                if (child instanceof Junkership) {
+                    if (Utility.hasCollision(this, child.hitBox)) {
+                        game.removeChild(this)
+                        this.destroy()
+                        child.score.incrementScore()
+                    }
+                }
+            })
+    }
+}
```

Comment:
Again the magic number. I would suggest using the form `Reference.JUNK_DESPAWN_RATE * 1000` because whatever number is used I'm sure we would prefer in terms of seconds instead of milliseconds.